foundation-extensions/mobile-core/README.md

```diff
@@ -1,34 +1,37 @@
 As of **April 1, 2020**, Apple will no longer support the UIWebView API. To avoid any issues, ensure that you are using iOS SDK version 2.3.4 or later. For more information about the UIWebView API, see [UIWebView ](https://developer.apple.com/documentation/uikit/uiwebview).
 {% endhint %}
 
+Mobile Core is mandatory for all Adobe Experience Platform app implementations. Mobile Core contains a common set of functionality and frameworks, including Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, and disk access routines. The following sections provide additional information about what is provided in the Mobile Core extension.
+
 ## Mobile Core generic APIs
 
 The SDK provides core, extension-independent APIs to facilitate the event tracking of user screens, actions, and PII data. Events generated by these APIs are published to the SDK event hub and are available for use by extensions. For instance, when the Analytics extension is installed, all user actions and app screens event data is sent to the appropriate Analytics reporting endpoints.
 
-For more information, see [Mobile Core API Reference](mobile-core-api-reference.md) and [Mobile Core Event Reference](mobile-core-event-reference.md).
+For more information, please read the [Mobile Core API reference](mobile-core-api-reference.md).
 
 ## Configuration
 
-The configuration extension provides the events in the Event Hub that provide the configuration details for all installed extensions. For more information, see [Configuration](configuration/).
+The configuration extension provides different APIs to set up your application's configuration settings. For more information, please read the documentation on [configuration](configuration/).
 
 ## Identity
 
-The Identity framework enables the Experience Cloud ID \(ECID\) service and is a required component for most SDK implementations. For more information, see [Identity](identity/).
+The identity framework lets your app use Experience Cloud ID (ECID). Using ECIDs improves synchronization with Adobe and other customer identifiers. Please read the documentation on the [identity framework](identity/) for more information.
 
 ## Lifecycle
 
-Sessions contain information about the app's current lifecycle, such as the device information, the application install or upgrade information, the session start and pause times, the number of application launches, and additional context data that is provided by the developer through the `LifecycleStart` API. Session data is persisted, so it is available across application launches. For more information, see [Lifecycle](lifecycle/).
+The Lifecycle extension lets you access information about the app's sessions, including device information, app installation, app upgrades, session start and pause times, number of application launches, and additional context data. For more information on how to use the Lifecycle API, please read the documentation on [Lifecycle](lifecycle/).
```

Comment:
why do we capitalize `Lifecycle` but not `identity`?

Reply:
btw, can we use `Identity extension` instead of `Identity framework` to keep the consistency ?

Reply:
I didn't realize that Identity was a proper noun in this context, and thought it was just a generic noun used. I'll capitalize identity if it's actually a proper noun usage here!